### ISSUE TITLE
Provide client examples

### ISSUE THREAD

**Issue by muesli** [ekanite/ekanite#85]
I'm trying to log to ekanite with logrus's syslog hook, but I can't get it working so far. Some examples for logging clients/libs would be nice.

**Comment by otoolep** [comment]
Are you sure syslog is configured as per the README? If so, then sending any data there should work. There are so many client libs out there, I concentrated on syslog, to which most clients can send data in their own way.

**Comment by muesli** [comment]
I think I realized logrus is following a different RFC and hence a slightly different encoding than ekanite. Which brings me back to the second part of my question: is there an "official" / designated golang client library for posting logs to ekanite?

**Comment by otoolep** [comment]
No, there is no such client. The whole point of Ekanite is that you send your logs to syslog or rsyslog-ng (or something) and configure syslog to send those logs to Ekanite.

**Comment by otoolep** [comment]
Though it would be pretty easy to create a client library that would emit log lines in the right format.
